react/hooks/useFetchMore.js

```diff
@@ -249,6 +273,8 @@ export const useFetchMore = props => {
     loading,
     from: pageState.from,
     to: pageState.to,
+    nextPage: pageState.nextPage,
+    previousPage: pageState.previousPage,
```

Comment:
On this file, I just intended to add those two variables to the return value, but ended up having to fix the lint issues 🥲